plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/QueryBuilder.java

```diff
@@ -170,6 +160,19 @@ protected String getProjection(List<JdbcColumnHandle> columns, Map<String, Strin
                 .collect(joining(", "));
     }
 
+    private String getFrom(JdbcRelationHandle baseRelation, Consumer<QueryParameter> accumulator)
```

Comment:
Would it be better if we make it `protected`. So that in future custom builder could directly use this `getFrom`. WDYT ?

Reply:
I thought about it, but finally decided to make it `private` as other methods except `getProjection` are `private` now. 
Maybe it worth to left it `private` for now and if one needs to `@Override` it - they can do this. It's harder to make `protected` method `private` again. Please let me know what do you think.

Reply:
I'm okay with it.